src/hotspot/share/gc/g1/g1CardSetMemory.hpp

```diff
@@ -84,44 +84,30 @@ typedef G1SegmentedArrayFreeList<mtGCCardSet> G1CardSetFreeList;
 // Since it is expected that every CardSet (and in extension each region) has its
 // own set of allocators, there is intentionally no padding between them to save
 // memory.
-template <class Slot>
 class G1CardSetAllocator {
```

Comment:
In the first sentence of the preceding block comment, "(Slot slots)" seems leftover from the template parameter.